include/GafferUIBindings/GadgetBinding.h

```diff
@@ -164,7 +164,7 @@ class GadgetWrapper : public GafferBindings::GraphComponentWrapper<WrappedType>
 					boost::python::object f = this->methodOverride( "doRenderLayer" );
 					if( f )
 					{
-						f( layer, GafferUI::StylePtr( const_cast<GafferUI::Style *>( style ) ) );
+						f( layer, GafferUI::StylePtr( const_cast<GafferUI::Style *>( style ) ), reason );
```

Comment:
Looks like this is throwing because we haven't bound `RenderReason` to python - see CI failures.